spec/rspec/support/encoded_string_spec.rb

```diff
@@ -209,9 +246,9 @@ module RSpec::Support
 
           it 'makes no changes to the resulting string' do
             resulting_array = build_encoded_string(non_ascii_compatible_string).split("\n")
-            expect(resulting_array.size).to eq(1) # sanity check
-            expected_string = non_ascii_compatible_string
-            expect_identical_string(resulting_array.first, expected_string, Encoding.find('UTF-16LE'))
+            expect(resulting_array).to match [
```

Comment:
Why do you call `match` here? Would you do that in other projects that use RSpec?